src/aind_data_transfer/jobs/transcode_job.py

```diff
@@ -261,5 +281,24 @@ def main():
         )
 
 
+
+    if job_configs["jobs"]["create_ng_link"]:
```

Comment:
This needs to occur before we upload the auxiliary files in the raw data dir, since it writes the json locally which gets uploaded with the other files.